addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1240,6 +1240,54 @@ registry.ImagesLazyLoading = publicWidget.Widget.extend({
     },
 });
 
+registry.BackgroundShape = publicWidget.Widget.extend({
+    selector: '.o_we_shape',
+
+    /**
+     * @override
+     */
+    start() {
+        this._onBackgroundShapeResize();
+        window.addEventListener('resize', this._onBackgroundShapeResize.bind(this));
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        window.removeEventListener('resize', this._onBackgroundShapeResize);
+        this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onBackgroundShapeResize() {
+        // Due to a bug on Chrome when using browser zoom, there is sometimes
+        // a gap between sections with shapes. This gap is due to a rounding
+        // issue when positioning the SVG background image. This code fixes
+        // that by ensuring that shape elements always have a width value as
+        // close to an integer as possible.
+        this.target.style.left = '';
+        this.target.style.right = '';
+        // Get the decimal part of the shape element width.
+        let decimalPart = this.target.getBoundingClientRect().width % 1;
+        // Round to two decimal places.
+        decimalPart = Math.round((decimalPart + Number.EPSILON) * 100) / 100;
```

Comment:
Why the use of `Number.EPSILON`?

Reply:
I saw this [here](https://stackoverflow.com/questions/11832914/how-to-round-to-at-most-2-decimal-places-if-necessary)